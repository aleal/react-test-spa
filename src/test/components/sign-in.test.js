import React from 'react';
import {renderComponent} from '../test-helper';
import SignIn from '../../components/sign-in';

describe('Sign In Tests' , () => {
    
    let component;

    beforeEach(() => {
        component = renderComponent(SignIn);
    });
  
    it('renders sign in page', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});