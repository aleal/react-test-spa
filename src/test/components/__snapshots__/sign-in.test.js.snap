// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign In Tests renders sign in page 1`] = `
<div
  className="container"
>
  <h3>
    Sign In
  </h3>
  <div
    className="form"
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className=""
      >
        <div
          className="text-help"
        >
          
        </div>
      </div>
      <div
        className="form-group "
      >
        <label>
          Email
        </label>
        <input
          className="form-control"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onDragStart={[Function]}
          onDrop={[Function]}
          onFocus={[Function]}
          type="text"
          value=""
        />
        <div
          className="text-help"
        >
          
        </div>
      </div>
      <div
        className="form-group "
      >
        <label>
          Password
        </label>
        <input
          className="form-control"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          onDragStart={[Function]}
          onDrop={[Function]}
          onFocus={[Function]}
          type="password"
          value=""
        />
        <div
          className="text-help"
        >
          
        </div>
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Submit
      </button>
      <a
        className="btn btn-danger"
        href="/"
        onClick={[Function]}
      >
        Cancel
      </a>
    </form>
  </div>
</div>
`;
